src/test/VesperAdapterV1.t.sol

```diff
@@ -159,6 +167,47 @@ contract VesperAdapterV1Test is DSTestPlus {
         TokenUtils.safeApprove(alUsdAddress, address(alchemistUSD), 2**256 - 1);
         TokenUtils.safeApprove(DAI, 0x43b4FdFD4Ff969587185cDB6f0BD875c5Fc83f8c, 2**256 - 1);
         hevm.stopPrank();
+
+        resolver = new HarvestResolver();
+        harvester = new AlchemixHarvester(address(this), 100000e18, address(resolver));
+        resolver.setHarvester(address(harvester), true);
+        hevm.startPrank(ADMIN);
+        alchemistUSD.setKeeper(address(harvester), true);
+        alchemistETH.setKeeper(address(harvester), true);
+        hevm.stopPrank();
+
+        resolver.addHarvestJob(
+            true,
+            vaDAI,
+            alchemistUSDAddress,
+            address(rewardCollectorVesperUSD),
+            address(vspRewardToken),
+            0,
+            0,
+            100
+        );
+
+        resolver.addHarvestJob(
+            true,
+            vaUSDC,
+            alchemistUSDAddress,
+            address(rewardCollectorVesperUSD),
+            address(vspRewardToken),
+            0,
+            0,
+            100
+        );
+
+        resolver.addHarvestJob(
+            true,
+            vaETH,
+            alchemistETHAddress,
+            address(rewardCollectorVesper),
+            address(vspRewardToken),
+            0,
+            0,
+            100
+        );
```

Comment:
since we're deploying a brand new harvester and resolver, lets add all the current jobs in this script to make life easer on ops squad